Assets/Scripts/Utility/RMBLayout.cs

```diff
@@ -1001,7 +1001,7 @@ private static bool IsBulletinBoard(uint modelID)
             return modelID == BulletinBoardModelID;
         }
 
-        private static void AddAnimalAudioSource(GameObject go, int record)
+        public static void AddAnimalAudioSource(GameObject go, int record)
```

Comment:
This method should be moved outside RMBLayout since it is now also used for RDB layout.

Reply:
@df-ferital Do you have any suggestions? I tried looking a bit, and couldn't find a good place. Maybe `GameObjectHelper`?

Reply:
Yes, I suppose `GameObjectHelper` would be the best place.

Reply:
I am new to the DFU scene. This change you're discussing - is it a change that I should be making, or is it something that more experienced DFU programmers will make after the pull request is merged? If I'm the one that should be making this change, should I cancel this pull request and re-do it? Thanks in advance for your time!

Reply:
You should do it, but you don't have to cancel. You can make a new commit, push the commit to the same branch (jeniferirwin:master), and the PR will update accordingly

Reply:
I've made the change and added the commit. Thanks for the explanation!